modules/kubernetes/logs/scrubs/json-nulls.river

```diff
@@ -0,0 +1,32 @@
+/*
+Module: scrub-json-nulls
+Description: Removes any json properties with a null value
+Arguments:
+  forward_to: Module to forward the output to
+*/
+argument "forward_to" {
+  optional = false
+}
+
+export "process" {
+  value = loki.process.scrub_json_nulls
+}
+
+loki.process "scrub_json_nulls" {
+  forward_to = [argument.forward_to.value]
+
+  // check logs.agent.grafana.com/scrub-nulls annotation, if true remove any json property whose value is set to null
```

Comment:
Ditto on moving this to a comment to explain why

Reply:
Updated description